src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Skia.Gtk/Package.appxmanifest

```diff
@@ -0,0 +1,43 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please revert this... these are copied automatically from the Windows head for both GTK & Linux when the template is packed